single-line-diagram-iidm-extensions/src/test/resources/branchStatusRef.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<iidm:network xmlns:iidm="http://www.powsybl.org/schema/iidm/1_5" xmlns:bs="http://www.powsybl.org/schema/iidm/ext/branch_status/1_0" id="test" caseDate="2016-06-27T12:27:58.535+02:00" forecastDistance="0" sourceFormat="test">
+    <iidm:substation id="S" country="FR">
+        <iidm:voltageLevel id="VL" nominalV="400.0" topologyKind="NODE_BREAKER">
+            <iidm:nodeBreakerTopology></iidm:nodeBreakerTopology>
+        </iidm:voltageLevel>
+    </iidm:substation>
+    <iidm:substation id="S2" country="FR">
+        <iidm:voltageLevel id="VL2" nominalV="400.0" topologyKind="NODE_BREAKER">
+            <iidm:nodeBreakerTopology></iidm:nodeBreakerTopology>
+        </iidm:voltageLevel>
+    </iidm:substation>
+    <iidm:line id="L" r="1.0" x="1.0" g1="0.0" b1="0.0" g2="0.0" b2="0.0" node1="2" voltageLevelId1="VL" node2="0" voltageLevelId2="VL2"/>
+    <iidm:extension id="L">
+        <bs:branchStatus status="PLANNED_OUTAGE"/>
```

Comment:
<bs:branchStatus>PLANNED_OUTAGE</bs:branchStatus> would be nicer?

Reply:
Why would it be nicer?

Reply:
To not repeat status?

Reply:
here we have something like <status status⁼''">

Reply:
Ok. Modification done.